### ISSUE TITLE
Pages between sites not connecting

### ISSUE THREAD

**Issue by mikem33** [lloc/Multisite-Language-Switcher#134]
I am experiencing problems with linking pages or posts between sites.
I recently have made a new personal WordPress site and I managed myself to link the sites with some problems but finaly I think I got it working. Recently, I noticed that the pages of the main site in the Network was not linked with the secondary site but the second site have its pages correctly linked. 
I have tried to relink a page in the main site with its respective page in the secondary site but this action did not work and unlinked the page in the secondary site.
The "Update" button in pages with Gutenberg seems not working well and if I use the main page Update button after selecting the desired page to link, when I refresh the page, the connection has not been made.
I also have tried to making the connection manually in the database and I get the same results, the pages in the second site remains linked before I inserted a row but in the original site this did not work.
I have used this plugin multiple times before Gutenberg came and I never had this problem before. Any idea about what is causing the problem?

**Comment by lloc** [comment]
OK, let's find the problem ... Do you use the select-inputs or the autocompletes? Are there any notices or warnings in your log-files?

**Comment by mikem33** [comment]
I'm sorry about the delay in answering this, but I was very busy and I kind of forgot of this opened issue. I have solved it now but I am not sure what was the problem with that because it was working fine when I uploaded the project to production. Maybe it was something related to the site language files because it was the last thing I did before it worked again. 
Anyway, thank you for your help and sorry again for the delay in answering.
